### ISSUE TITLE
Windows: Platform.recordingsDir doubles SuperCollider directory

### ISSUE THREAD

**Issue by jrsurge** [supercollider/supercollider#4412]
## Environment

* SuperCollider version: 3.10.2
* Operating system: Windows 10, 1809
* Other details (Qt version, audio driver, etc.): N/A

## Steps to reproduce

```supercollider
Platform.recordingsDir;
```

## Expected vs. actual behavior

For consistency across platforms, should return:
```supercollider
"C:/Users/<username>/Documents/SuperCollider/Recordings"
```

Currently returns:
```supercollider
"C:/Users/<username>/Documents/SuperCollider/SuperCollider/Recordings"
```

From WindowsPlatform.sc:
```supercollider 
initPlatform {
    super.initPlatform;
    recordingsDir = this.myDocumentsDir +/+ "SuperCollider" +/+ "Recordings";
}

// ...

myDocumentsDir {
    _WinPlatform_myDocumentsDir
    ^this.primitiveFailed
}
```

I can submit a PR, but wanted to check if it's a change ```WindowsPlatform::initPlatform```, or the ```_WinPlatform_myDocumentsDir``` primitive? Whose responsibility is it to append the "SuperCollider" to the MyDocuments path?

**Comment by jamshark70** [comment]
This isn't a well-defined feature, so I'll make a judgment call: let's have --

- `myDocumentsDir` pointing to ".../Documents/SuperCollider/"

- `recordingsDir` appending "Recordings"

Also note that `myDocumentsDir` is not implemented for other platforms. Perhaps not relevant here but it isn't a safe public interface for cross-platform code, so we shouldn't advertise it too widely.

**Comment by snappizz** [comment]
fixed in #4420. thanks!
